source/jormungandr/tests/line_sections_tests.py

```diff
@@ -365,7 +369,7 @@ def test_route_schedule_impacted_by_line_section(self):
         r = self.query_region('lines/line:1/departures?{cur}&{d}&{f}'.format(cur=cur, d=dt, f=fresh))
         d = get_all_element_disruptions(r['departures'], r)
         assert 'line_section_on_line_1' in d
-        assert impacted_ids(d) == {'C_1', 'D_1', 'D_3', 'E_1', 'vj:1:1'}
+        assert impacted_ids(d) == {'vj:1:1'}
```

Comment:
you may want to add some test on "/lines/line:1" "/stop_point/stop_point:C_1" "vehicule_journeys/vehicle_journey:vj:1:1" to test if the line_section is displayed in the "global" display_information?

Reply:
vj id correspond to display_information

Reply:
I don't understand

Reply:
ok